language-design.md

```diff
@@ -27,7 +56,19 @@ How do the themes of _Growing a Language_ relate to the lab we did this week?
 
 **Response**
 
+In the sound lab there was the issue that each sound could only have a single transformation applied to it (and an unavoidable side effect of playing the sound and writing to a file).  This made it very hard to extend the library - multiple transformations couldn't be done by cleanly calling multiple methods sequentially, but instead by writing a new function that explicitly performed all of those transformations (or even worse, daisy-chaining using the fact that the output file name was "out.wav").  This violated [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) as well as ease of expansion and use.  
```

Comment:
In many languages, there is tons of repetition, sometimes in the syntax and sometimes in the semantics (for example, list.sort() and sorted(iterable). Is DRY always a bad thing or is it sometimes ok to include repetition?